### DISCUSSION TITLE
Videos are generally very laggy

### DISCUSSION THREAD

**Discussion by ggtylerr** [FreeTubeApp/FreeTube#5431] [Q&A]
I'm on a 1080p video, fullscreen, with a 3050 Ti. Freetube maxes out my GPU usage and still drops half of the frames:
![image](https://github.com/user-attachments/assets/66d09efa-82fe-47d4-a276-17ce70b1344b)
![image](https://github.com/user-attachments/assets/996eda27-d2f2-4c77-b945-5b1091da5bc9)
(This happens when both enabling and disabling DASH AV1.)

EDIT: It seems switching to 480p drops the usage to a more reasonable 20% usage with 0 dropped frames, so it seems DASH is the main culprit. (Although it should be noted that mpv and potplayer both take less resources, around 15%-20%, while on 1080p.)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> This happens when both enabling and disabling DASH AV1

Did you leave the watch page when you enabled/disabled this?

**Reply by ggtylerr** [reply]
Yes.

**Comment by absidue** [comment]
Can you reproduce this with the lastest release 0.22.0?
